_includes/tutorials/produce-consume-lang/scala/code/settings.gradle

```diff
@@ -1,13 +1,3 @@
-pluginManagement {
-    repositories {
-        gradlePluginPortal()
-        jcenter()
```

Comment:
@bbejeck only `gradlePluginPortal()` should be there. or just remove `jcenter()` and keep rest